README.md

```diff
@@ -19,192 +90,345 @@ $ make
-    * Common functions of Postgres and GridDB (same as for PostgreSQL): char_length, concat, lower, upper, substr, round, ceiling, ceil, floor.
-    * Unique function of Griddb: to_timestamp_ms, to_epoch_ms, array_length, element, timestamp, timestampadd, timestampdiff,       
-                                 time_next, time_next_only, time_prev, time_prev_only, time_interpolated, time_sampling, max_rows, min_rows, now.
-    * For the push down of now() and timestamp() function, please add prefix `griddb_` for these functions.     
-      Example: now() -> griddb_now()
+### Grant usage on foreign server to normal user in PostgreSQL:
 
-- Support LIMIT...OFFSET pushdown when having LIMIT clause only or both LIMIT and OFFSET.
-- Support ORDER BY pushdown with column, except ORDER BY column ASC NULLS FIRST, ORDER BY column DESC NULLS LAST and ORDER BY functions/formulas
-- Support aggregation push down:
-    * Common functions of Postgres and Griddb are pushed down: min, max, count(*), sum, avg, variance, stddev.
-    * Unique function of Griddb are pushed down: time_avg<br>
+Once for a normal user (non-superuser) in PostgreSQL, as PostgreSQL superuser. It is a good idea to use a superuser only where really necessary, so let's allow a normal user to use the foreign server (this is not required for the example to work, but it's secirity recomedation).
 
-For PostgreSQL version 14 or later:
-- Support bulk INSERT by using batch_size option.
-- Support keep connections open by use keep_connections option:
-    * keep_connections option that controls whether griddb_fdw keeps the connections to the foreign server open so that the subsequent queries can re-use them. This option can only be specified for a foreign server. The default is on. If set to off, all connections to the foreign server will be discarded at the end of transaction. losed connections will be re-established when they are necessary by future queries using a foreign table.
-- Support list cached connections to foreign servers by using function griddb_get_connection().
-- Support discard cached connections to foreign servers by using function griddb_disconnect('server_name'), griddb_disconnect_all().
-- Support deparse and push down ANY/ALL ARRAY with the argument is constant.
+```sql
+	GRANT USAGE ON FOREIGN SERVER sqlite_server TO pguser;
+```
+Where `pguser` is a sample user for works with foreign server (and foreign tables).
+
+### User mapping
+
+Create an appropriate user mapping:
+```sql
+    	CREATE USER MAPPING
+	FOR pguser
+	SERVER griddb_svr 
+    	OPTIONS (
+	  username 'username',
+	  password 'password'
+	);
+```
+Where `pguser` is a sample user for works with foreign server (and foreign tables).
+
+### Create foreign table
+All `CREATE FOREIGN TABLE` SQL commands can be executed as a normal PostgreSQL user if there were correct `GRANT USAGE ON FOREIGN SERVER`. No need PostgreSQL supersuer for secirity reasons but also works with PostgreSQL supersuer.
```

Comment:
Please fix the typo 'secirity'?

Reply:
Done. "u" and "i" is near on keyboard ...

Reply:
Thank you for support.